miden/benches/deserialize_std_lib.rs

```diff
@@ -9,8 +8,7 @@ fn deserialize_std_lib(c: &mut Criterion) {
     group.measurement_time(Duration::from_secs(15));
     group.bench_function("read_from_bytes", |bench| {
         bench.iter(|| {
-            let _ =
-                Library::read_from_bytes(StdLibrary::SERIALIZED).expect("failed to read std masl!");
```

Comment:
I would still keep an explicit call to `Library::read_from_bytes()` in the benchmark - we could probably even copy the implementation in `StdLibrary::default()`. This is so that

1. it's what function we're benchmarking
2. If we change the default implementation, the benchmark is unaffected
3. I don't want to have to remind myself the specifics of `bench.iter()` to ensure that subsequent calls to `StdLibrary::default()` aren't just reading the cached value (I'd be very surprised if they were, but I'm not 100% sure without looking it up)